docs/zh_CN/FreeBSD.md

```diff
@@ -0,0 +1,77 @@
+# Minio FreeBSD Quickstart Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+### Minio with ZFS backend - FreeBSD
+This example assumes that you have a FreeBSD 11.x running
```

Comment:
Will this also be translated?

Reply:
Yes,right now I only translated two files,the others in zh_CN is still in english,I will finish the job in october.